server/routes/establishments/bulkUpload/validate/validateWorkerCsv.js

```diff
@@ -60,16 +67,19 @@ const validateWorkerCsv = async (
   const thisWorkerAsAPI = lineValidator.toAPI();
 
   try {
-    const foundCurrentEstablishment = myCurrentEstablishments.find(
-      (establishment) => establishment.key === lineValidator.establishmentKey,
-    );
-    const foundCurrentWorker = foundCurrentEstablishment.theWorker(lineValidator.key);
-
-    if (thisWorkerAsAPI.postcode && foundCurrentWorker && foundCurrentWorker.postcode !== thisWorkerAsAPI.postcode) {
-      const { Latitude, Longitude } = (await postcodes.firstOrCreate(thisWorkerAsAPI.postcode)) || {};
+    if (thisWorkerAsAPI.status === 'NEW') {
+      if (thisWorkerAsAPI.postcode) {
```

Comment:
Could combine this `if`?

```suggestion
    if (thisWorkerAsAPI.postcode && thisWorkerAsAPI.status === 'NEW') {
```

Reply:
No, because if you're new, you shouldn't run the else, even if you don't have a postcode